src/core/InputBase.tsx

```diff
@@ -0,0 +1,16 @@
+import { InputBase as MuiInputBase } from '@material-ui/core'
+import React, { SFC } from 'react'
+import { IDefault } from './Advertise'
+
+export interface IProps extends IDefault {
+    children: React.ReactNode
```

Comment:
addd another props, according to https://material-ui.com/api/input-base/

Reply:
Solved